README.md

```diff
@@ -16,6 +16,109 @@ The Wavefront Java Tracing Agent provides application observability without havi
 The Wavefront Java Tracing Agent includes:
 * The [Java OpenTracing SpecialAgent](https://github.com/opentracing-contrib/java-specialagent), which automatically instruments Java applications with traces.
 * The Wavefront OpenTracing Bundle that is implemented in this repository, which sends the tracing data to Wavefront for observability.
+
+**Before you start implementing, let us make sure you are using the correct SDK!**
+
+![Java Tracing Agent Decision Tree](docs/java_tracing_agent.png)
+
+> <b><i>Note</i></b>:
+> </br>
+>   * **This is the Wavefront by VMware Java Tracing Agent!**
+>   If this is not what you were looking for, see the [table](#wavefront-sdks) given below.
+>   * Want to write code to instrument your application? Use the [Wavefront Java OpenTracing SDK](https://github.com/wavefrontHQ/wavefront-opentracing-sdk-java).
```

Comment:
For the second bullet, I suggest: 
* Want control over instrumenting your code? Use the... 

(Nobody really wants to write code, but maybe the agent doesn't offer enough control)